sdk/python/setup.py

```diff
@@ -24,28 +24,28 @@
 REQUIRES_PYTHON = ">=3.6.0"
 
 REQUIRED = [
-    "Click==7.*",
-    "google-api-core==1.20.*",
-    "google-auth<2.0dev,>=1.14.0",
-    "google-cloud-bigquery==1.18.*",
-    "google-cloud-storage==1.20.*",
-    "google-cloud-core==1.0.*",
-    "googleapis-common-protos==1.*",
-    "google-cloud-bigquery-storage==0.7.*",
-    "grpcio==1.*",
-    "pandas~=1.0.0",
-    "pandavro==1.5.*",
-    "protobuf>=3.10",
-    "PyYAML==5.1.*",
-    "fastavro>=0.22.11,<0.23",
-    "kafka-python==1.*",
-    "tabulate==0.8.*",
-    "toml==0.10.*",
-    "tqdm==4.*",
-    "pyarrow<0.16.0,>=0.15.1",
-    "numpy",
-    "google",
-    "confluent_kafka",
+    "click>=7.1.2",
```

Comment:
Are these really the lowest versions we can use?

Reply:
They are the latest versions we can use instead. This way the dependencies will not be outdated for longer, but is likely going to conflict with old packages. Although no conflicts are detected when I last tested the dependencies by installing in a new `virtualenv` with `tensorflow==2.3.0`.

Reply:
Yea but basically, if somebody is running Pandas 1.0.0 then Feast wont work for them.

Reply:
@SwampertX I agree with @woop, the dependency requirements should be as low as possible since many of our existing users are using older version of library.